modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -387,9 +408,9 @@ function filtered_postgresql_restore {
     single_transaction=''
   fi
 
-  output_restore_sql \
-    | sudo psql -U aws_db_admin -h "${database_hostname}" "${single_transaction}" \
-      --no-password -d "${database}" 2>&1
+  output_restore_sql
+  sudo psql -U aws_db_admin -h "${database_hostname}" "${single_transaction}" \
+    --no-password -d "${database}" -f "${tempdir}/restore.sql" 2>&1
```

Comment:
I wonder if life would be easier if we change pg_dump to not be format custom and just output SQL, then we'd not need to use pg_restore at all?

It'd be a breaking change but we could just skip a day of db pulling to populate the new databases.

Reply:
I guess we'd have an issue with file sizes, since `pg_dump --format=plain` doesn't do any compression.

We could compress them with gzip before uploading them to S3, but we'd have to be careful to avoid the same issue we're having here with piping large amounts of data from docker.

Maybe we'd be better off building a debian package for postgres13 and sacking off docker after all...

Reply:
Would agree that having the package installed is best, not sure how much of a time sink that would be.

We take the plain and gzip approach already for MySQL (and I think other dump formats) we could always unzip the file first and put it in a temp file. That's essentially what this change is doing except it's using pg_restore as an agent to decompress the SQL.